eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -139,21 +138,21 @@ conwayGovCertTransition = do
     judgmentContext
   case c of
     ConwayRegDRep cred _deposit _mAnchor -> do
-      Set.notMember cred vsDReps ?! ConwayDRepAlreadyRegistered cred
+      Map.notMember cred vsDReps ?! ConwayDRepAlreadyRegistered cred
       -- TODO: check against a new PParam `drepDeposit`, once PParams are updated. -- someCheck ?! ConwayDRepIncorrectDeposit deposit
-      pure $ vState {vsDReps = Set.insert cred vsDReps}
+      pure $ vState {vsDReps = Map.insert cred undefined vsDReps}
```

Comment:
Are we gonna leave this as undefined?  Shouldn't we build a `DRepState` using the given anchor?

Reply:
oops, nice catch!